app/js/controllers/main.js

```diff
@@ -3,12 +3,11 @@
 export default class MainController extends Controller {
   constructor(options) {
     super(options);
-    this.attached = false;
-    this.waitToBeOpened();
+    this._waitToBeOpened();
   }
 
-  waitToBeOpened() {
-    var openGesture = {
+  get openGesture() {
```

Comment:
The gesture control code here is big enough that it might be overwhelming ```MainController```. That is, instead of ```MainController``` just tying everything together, it has a significant amount of body logic in it. Maybe we should move this into a new ```GestureService``` or ```GestureController```. This isn't a strong opinion though.